docs.md

```diff
@@ -23,72 +23,82 @@ permalink: /docs/
 
 ## New to Flutter?
 
-* [Write Your First Flutter App](/get-started/codelab/)  
-  A walkthrough on creating a Flutter app that generates names for startup companies.
+Start here.
```

Comment:
This is confusing when we have "Get started" above. Maybe we want to say something like "once you've gone through Get Started [LINK], here are the immediate next steps" or something like that. And this means we drop "Write Your First Flutter App" from here since that's included in the Get Started flow (and should be almost a requirement).

Reply:
Done and Done.